lib/fixer.js

```diff
@@ -2,7 +2,7 @@ var isValidSemver = require('semver/functions/valid')
 var cleanSemver = require('semver/functions/clean')
 var validateLicense = require('validate-npm-package-license')
 var hostedGitInfo = require('hosted-git-info')
-var isBuiltinModule = require('is-core-module')
+var moduleBuiltin = require('module')
```

Comment:
```suggestion
var moduleBuiltin = require('node:module')
```

We're slowly moving over to use this exclusively, so we change it when we see it.

Reply:
ah, thought older stuff was preferred, considering the code still uses var :P should i also use destructuring to make the import less verbose?